<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatGroup</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/res.qrc">
    <normaloff>:/cloud.png</normaloff>:/cloud.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_14">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>3</number>
        </property>
        <item>
         <widget class="QTableWidget" name="UsrListTableWidget">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>250</number>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="70,30">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8" stretch="10,120">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="3,7">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QLabel" name="UsrNameLabel">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <widget class="QLabel" name="GroupNameLabel">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QTextBrowser" name="ChatTextBrowser"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6" stretch="10,150,0">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="10,10,10,500">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QPushButton" name="CapturePushButton">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="resources/res.qrc">
                  <normaloff>:/capture.png</normaloff>:/capture.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>35</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QPushButton" name="TranFilePushButton">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="resources/res.qrc">
                  <normaloff>:/open.png</normaloff>:/open.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>35</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QPushButton" name="EmojiPushButton">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="resources/res.qrc">
                  <normaloff>:/emoji.png</normaloff>:/emoji.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>35</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="ToolLabel">
                <property name="frameShape">
                 <enum>QFrame::Box</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QTextEdit" name="ChatTextEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="80,20">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string> </string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QPushButton" name="SendPushButton">
                <property name="text">
                 <string>Send(S)</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="StartMenudd">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>144</y>
      <width>195</width>
      <height>100</height>
     </rect>
    </property>
    <property name="title">
     <string>Start(F1)</string>
    </property>
    <addaction name="actionlogin"/>
    <addaction name="actionlogout"/>
   </widget>
   <addaction name="StartMenudd"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionlogout">
   <property name="icon">
    <iconset resource="resources/res.qrc">
     <normaloff>:/logout.png</normaloff>:/logout.png</iconset>
   </property>
   <property name="text">
    <string>logout</string>
   </property>
  </action>
  <action name="actionlogin">
   <property name="icon">
    <iconset resource="resources/res.qrc">
     <normaloff>:/login.png</normaloff>:/login.png</iconset>
   </property>
   <property name="text">
    <string>login</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
